src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -179,6 +214,23 @@ export default {
         this.currentWell.ccs_analysis_output = 'No'
       }
     },
+    updateAdaptiveLoading() {
+      if (this.currentWell.loading_target) {
+        this.currentWell.use_adaptive_loading = 'True'
+      } else {
+        this.currentWell.use_adaptive_loading = 'False'
+      }
+    },
+    formatLoadingTargetValue(val) {
+      if (val) {
+        if (this.regexToMatch.test(val)) {
+          return val
```

Comment:
See if you had `abc.45` in the val, that's what you'd get back from this format function because it matches the regex.  If you want help forming a better regex, let me know.  I've got a lot of experience with them.